tests/automated/sys/net/gnrc/network_layer/ipv6/suite-01_init/Makefile

```diff
@@ -0,0 +1,18 @@
+APPLICATION = ipv6
+#include ../Makefile.tests_common
+BOARD ?= native
+RIOTBASE ?= $(CURDIR)/../../../../../../../../
```

Comment:
Me too. At least it seems awkward.
Writing that line, I was curious if someone may show me that RIOTs make system provides a better method to examine the project root.
As `$(CURDIR)/../../../../../../../../` and `$(CURDIR)/../..` (in tests Makefile.tests_common) is the same Semantic just with a different number of "/..", what would you say? How many subdirs are acceptable?